### PR TITLE
[13.0][ADD] stock_picking_package_grouped: New module to create package from pickings depending on grouping criteria

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT28843

ping @victoralmau @carlosdauden

### CODE REVIEW

File: stock_picking_package_grouped/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Picking Package Groups",
+    "summary": "Allow create package (put in pack) from pickings "
+    "depending on grouping criteria.",
+    "version": "13.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
```
Comment (victoralmau) [inline stock_picking_package_grouped/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/stock-logistics-warehouse",
    "maintainers": ["sergio-teruel"],
```

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Comment by sergio-teruel** [comment]
@victoralmau Changes done!!

File: stock_picking_package_grouped/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Sergio Teruel - Tecnativa
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class PickingType(models.Model):
```
Comment (rousseldenis) [inline stock_picking_package_grouped/models/stock_picking.py]: Could you put one model per file ?

**Review by rousseldenis — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@rousseldenis Changes done!!

**Pull request merged**